media-radio/KochMorse/KochMorse-3.4.0.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6	# cmake-utils.eclass does not support EAPI 7
+
+inherit cmake-utils
+
+DESCRIPTION="Morse-code tutor using the Koch method"
+HOMEPAGE="https://github.com/hmatuschek/kochmorse"
+SRC_URI="https://github.com/hmatuschek/kochmorse/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND="dev-qt/linguist-tools
```

Comment:
You can sort these for easier readability. Also, styling preference, but you can do the following in here: 
```
RDEPEND="dev-qt/qtmultimedia"
DEPEND=${RDEPEND}
	dev-util/cmake[qt5]
	dev-qt/linguist-tools"
```
(but in the end it doesnt make a difference, just letting you know RDEPEND can be defined before DEPEND)